### ISSUE TITLE
The training accuracy of imagenet data set is not up to standard

### ISSUE THREAD

**Issue by cyx2016** [frank-xwang/RIDE-LongTailRecognition#34]
I conducted the three expert training according to the readme method, and the training result of ImageNet-LT can only reach about 51.8%. May I ask why?
![image](https://user-images.githubusercontent.com/17540452/222022245-d948e438-ada6-410a-bc5c-b0cff05dbe19.png)

**Comment by frank-xwang** [comment]
Hi, could you please share your training settings? If you use the same training commands and settings (batch size, learning rate, epochs, etc) as we did, you should be able to reproduce our results, and in many cases, even achieve better performance than what was reported in our paper. We have tested the reproducibility of our model on local runs, and it matches the reported results. Furthermore, many follow-up works such as SADE [NeurIPS 2022], LTR [CVPR 2022], and ACE [ICCV 2021], have used RIDE, and the reproducibility has been confirmed by them.

**Comment by cyx2016** [comment]
The configuration I use is the configuration file config_imagenet_lt_resnet50_ride.json, the only difference is that I use one GPU instead of eight
![image](https://user-images.githubusercontent.com/17540452/222365139-28b3ff2d-9115-45d4-9bb0-eb9b23e001fb.png)

**Comment by frank-xwang** [comment]
Hi! I think you are using the wrong config file. The results in the paper are produced with [config_imagenet_lt_resnet50_distill_ride.json](https://github.com/frank-xwang/RIDE-LongTailRecognition/blob/main/configs/config_imagenet_lt_resnet50_distill_ride.json). Without distillation, the performance should be approximately 1% worse.

To further assist you in resolving the reproducibility issue, could you please provide us with the training log file as well? As we did not experience this issue during our own experiments with the default settings (8 or 4 GPUs), it is difficult to identify the issue without additional information. 

Additionally, since you are using only one GPU, I guess you have to decrease the batch size. We recommend adjusting the learning rate accordingly. A smaller batch size may require a smaller learning rate to achieve optimal results.

**Comment by cyx2016** [comment]
Thank you very much for your reply. My batch is indeed set small. I will try to set batch larger and use 4 GPUs or modify the learning rate。The result of the paper is that distillation is used, right? If not, the accuracy will be reduced by about 1%, right? I still want to confirm this。

**Comment by frank-xwang** [comment]
Yes, that's correct.

**Comment by cyx2016** [comment]
Thank you very much for your reply
